crates/core/component/dex/src/component/position_manager.rs

```diff
@@ -390,6 +392,10 @@ pub trait PositionManager: StateWrite + PositionRead {
             }
         }
 
+        // Update the candlestick tracking
+        self.record_position_execution(&prev_state, &new_state, &context)
+            .await?;
```

Comment:
Can we swallow the error here please, we don't want failure to record auxiliary data to snowball into a chain halt

Reply:
We could map the error and use `tracing::warn!`